webpages/settings/components/category-selector.html

```diff
@@ -68,8 +68,7 @@
     transition: all 0.2s ease;
     display: block;
     width: 4px;
-    border-radius: 0 4px 4px 0;
-    height: 40px;
+    height: 100%;
     background: var(--orange);
```

Comment:
This could also be changed to `--brand-orange` now that we don't have to worry about icon-recoloring (#6240).

Reply:
just got unbanned, also i dont wanna make @redguy12 kill me

Reply:
> just got unbanned, also i dont wanna make @RedGuy12 kill me

ask redguy12 why he'll never stop yapping at me if i agree with this suggestion